docs/competition/virtual-competitions/competition-session-checklist.md

```diff
@@ -0,0 +1,79 @@
+# Competition Session Checklist
+
+This checklist enumerates the tasks that need to be completed to run a virtual competition session with a module announcement.
+
+## Pre Event
+
+- Advertise the code submission deadline to teams
+- Create the YouTube livestream
+- Update the event page on the website with the correct livestream URL
+- Create a folder for session uploads in Google Drive
+- Create a presentation detailing the new module
+- Assign roles for this session
+    - Match Runners - Run the simulations of the matches
+    - Presenters - Present and commentate the livestream
+    - Producer - Runs the technical aspects of the livestream
+    - Scorer - Updates the compstate with match files
+    - Chat Mods - Monitor the livestream and Discord chats
+- Check for any new dropouts and update the compstate as necessary
+- Presenters prepare a scripted introduction to the session
+- Revise the rules for the new module
+- Update the simulator for the new module in the private repo
+- Draft a post-event website article with details of the new module (any game sensitive content to be reviewed via Google Drive)
+
+## After the Code Submission Deadline
+
+- Check for any no-shows
+- Construct the league for the session
+- Import the league into the compstate
+
+## Match Running
+
+- Get a copy of the updated compstate
+- Run the simulations
+- Transcode the outputted video files to WebM
+- Upload the entire output to our Google Drive Folder
+
+## Pre Go Live
+
+- Check the compbox services are still running
+- Put the website into competition mode
+- Producer downloads the recordings and configures the video scheduler to find the files correctly
+- Scorer downloads the match files
+- Presenters rehearse any presentations they need to give
+- Presenters join a pre-livestream call to rehearse their interactions
+
+## Go Live
+
+- Presenters present
+- Scorer commits and pushes match files to the compstate as they are broadcast
+- Chat Mods to monitor chat and forward on any relevant messages to presenters or producer via Slack (the `@commentators` group may be useful for this)
+
+## Module Announcement
+
+- Present the prepared presentation
+- Chat Mods collect questions from the livestream and Discord then paste into a Slack channel
+- Available volunteers comment on questions (in threads) with any answers or thoughts
+- Presenters read the questions, any answers in the thread, and add their own comments
+- Release the rules
+- Release the simulator update
+- Publish the prepared website article with the module announcement
+- End the livestream
+
+## Post Livestream
+
+- Cut down the VOD on YouTube to remove the countdown
+- Add chapters to the VOD
+- Distribute logs and animation files to teams
+- Schedule a Retro after a brief break
+- Announce the final scores, VOD, and next competition time to the teams
+
+## Retro
+
+- Have a Retro
+- Record actions to be taken
+
+## Post Retro
+
+- Revert the website back to non competition mode
```

Comment:
This doesn't need to happen after the retro. It should however happen after the news post has been published.

Reply:
https://github.com/srobo/runbook/pull/98/commits/7d409a6246fc5b096081a1b7309929a631c31a36